tools/data-api/models/service_version.go

```diff
@@ -0,0 +1,34 @@
+package models
+
+type ServiceVersionDetails struct {
+	// Resources is a key (Resource Name) value (ResourceSummary) pair of Resource
+	// supported by this Service Version
+	Resources map[string]ResourceSummary `json:"resources"`
+
+	// Source specifies the original source used for these API Definitions
+	Source ApiDefinitionsSource `json:"source"`
+}
+
+type ResourceSummary struct {
+	// OperationsUri is an endpoint which contains information about the Operations
+	// supported by this Resource
+	OperationsUri string `json:"operationsUri"`
+
+	// SchemaUri is an endpoint which contains information about the Schema for
+	// this Resource
+	SchemaUri string `json:"schemaUri"`
+}
+
+// ApiDefinitionsSource is used to indicate the original source for these API Definitions.
+type ApiDefinitionsSource string
+
+const (
+	// ApiDefinitionsSourceHandWritten is used to signify that this set
+	// of API Definitions was created by hand.
+	ApiDefinitionsSourceHandWritten ApiDefinitionsSource = "HandWritten"
+
+	// ApiDefinitionsSourceResourceManagerRestApiSpecs is used to signify
+	// that this set of API Definitions is based on data within the Azure
+	// Rest API Specs repository.
+	ApiDefinitionsSourceResourceManagerRestApiSpecs ApiDefinitionsSource = "ResourceManagerRestApiSpecs"
```

Comment:
we'll also need to add the Graph version here:

```suggestion
	ApiDefinitionsSourceHandWritten ApiDefinitionsSource = "HandWritten"

	// ApiDefinitionsSourceResourceManagerRestApiSpecs is used to signify
	// that this set of API Definitions is based on data within the Azure
	// Rest API Specs repository.
	ApiDefinitionsSourceResourceManagerRestApiSpecs ApiDefinitionsSource = "ResourceManagerRestApiSpecs"

	// ApiDefinitionsSourceResourceManagerMicrosoftGraphMetadata is used to signify
	// that this set of API Definitions is based on data within the Microsoft Graph
	//  Metadata repository.
	ApiDefinitionsSourceMicrosoftGraphMetadata ApiDefinitionsSource = "MicrosoftGraphMetadata"
```